tests/unit/test-class-wc-payments-express-checkout-button-helper.php

```diff
@@ -0,0 +1,152 @@
+<?php
+/**
+ * These tests make assertions against class WC_Payments_Express_Checkout_Button_Helper.
+ *
+ * @package WooCommerce\Payments\Tests
+ */
+
+use WCPay\Duplicate_Payment_Prevention_Service;
+use WCPay\Payment_Methods\CC_Payment_Method;
+use WCPay\Session_Rate_Limiter;
+
+/**
+ * WC_Payments_Payment_Request_Button_Handler_Test class.
+ */
+class WC_Payments_Express_Checkout_Button_Helper_Test extends WCPAY_UnitTestCase {
+	/**
+	 * Used to get the settings.
+	 *
+	 * @var WC_Payment_Gateway_WCPay|MockObject
+	 */
+	private $mock_wcpay_gateway;
+
+	/**
+	 * WC_Payments_Account instance.
+	 *
+	 * @var WC_Payments_Account
+	 */
+	private $mock_wcpay_account;
+
+	/**
+	 * Express Checkout Helper instance.
+	 *
+	 * @var WC_Payments_Express_Checkout_Button_Helper
+	 */
+	private $express_checkout_helper;
+
+	/**
+	 * Sets up things all tests need.
+	 */
+	public function set_up() {
+		parent::set_up();
+
+		$this->mock_wcpay_account = $this->createMock( WC_Payments_Account::class );
+		$this->mock_wcpay_gateway = $this->make_wcpay_gateway();
+
+		$this->mock_express_checkout_helper = new WC_Payments_Express_Checkout_Button_Helper( $this->mock_wcpay_gateway, $this->mock_wcpay_account );
```

Comment:
Just a tiny typo, this is definitely non-blocking, thanks for having a look 🙏 


```suggestion
		$this->express_checkout_helper = new WC_Payments_Express_Checkout_Button_Helper( $this->mock_wcpay_gateway, $this->mock_wcpay_account );
```

Reply:
Nice catch! I missed this note before merging! I'll make sure it's updated 👍